Frontend/ui-customer/src/Screens/CustomerUtils/MyProfileScreen/Controller.purs

```diff
@@ -128,10 +129,10 @@ eval (UserProfile (GetProfileRes profile)) state = do
   continue state { data { name = name, editedName = name, gender = gender, emailId = profile.email } }
 eval (NameEditTextAction (PrimaryEditText.TextChanged id value)) state = do
   _ <- pure $ spy "Value changed"  value
-  continue state { data { editedName = value }, props{genderOptionExpanded = false, expandEnabled = false, isBtnEnabled = ((length value >=3) && state.props.isEmailValid )} }
+  continue state { data { editedName = value }, props{genderOptionExpanded = (state.props.fromHomeScreen || false), expandEnabled = (state.props.fromHomeScreen || false), isBtnEnabled = if (state.props.fromHomeScreen) then false else ((length value >=3) && state.props.isEmailValid )} }
 eval (EmailIDEditTextAction (PrimaryEditText.TextChanged id value)) state = do
-  if (value == "" && state.data.errorMessage == Just EMAIL_EXISTS) then continue state {props{ isEmailValid = false, isBtnEnabled = false, genderOptionExpanded = false, expandEnabled = false}}
-    else continue state {data {editedEmailId = Just value , errorMessage = if (length value == 0) then Nothing else if ( validateEmail value) then Nothing else Just INVALID_EMAIL },props{ isEmailValid = if (length value == 0) then true else validateEmail value, isBtnEnabled = (length state.data.editedName >=3)  && (if (length value == 0) then true else validateEmail value), genderOptionExpanded = false}}
+  if (value == "" && state.data.errorMessage == Just EMAIL_EXISTS) then continue state {props{isEmailValid = false, isBtnEnabled = false, genderOptionExpanded = (state.props.fromHomeScreen || false), expandEnabled = (state.props.fromHomeScreen || false)}}
```

Comment:
Remove this `|| false` in all cases anyways it is depend on the `state.props.fromHomeScreen`.